presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkSessionProperties.java

```diff
@@ -145,7 +152,42 @@ public PrestoSparkSessionProperties(PrestoSparkConfig prestoSparkConfig)
                         prestoSparkConfig.getOutOfMemoryRetrySparkConfigs(),
                         true,
                         value -> MAP_SPLITTER.split(nullToEmpty((String) value)),
-                        value -> value));
+                        value -> value),
+                dataSizeProperty(
+                        SPARK_AVERAGE_INPUT_DATA_SIZE_PER_EXECUTOR,
+                        "Average input data size per executor",
+                        prestoSparkConfig.getAverageInputDataSizePerExecutor(),
+                       false),
+                integerProperty(
+                        SPARK_MAX_EXECUTOR_COUNT,
+                        "Maximum number of executors to run a query",
+                        prestoSparkConfig.getMaxExecutorCount(),
+                        false),
+                integerProperty(
+                        SPARK_MIN_EXECUTOR_COUNT,
+                        "Minimum number of executors to run a query",
+                        prestoSparkConfig.getMinExecutorCount(),
+                        false),
+                dataSizeProperty(
+                        SPARK_AVERAGE_INPUT_DATA_SIZE_PER_PARTITION,
+                        "Average input data size per partition",
+                        prestoSparkConfig.getAverageInputDataSizePerPartition(),
+                        false),
+                integerProperty(
+                        SPARK_MAX_HASH_PARTITION_COUNT,
+                        "Maximum hash partition required by the query",
```

Comment:
I usually keep the config description same for session property and its corresponding config.